gc/base/MemoryPoolAddressOrderedList.hpp

```diff
@@ -63,15 +63,24 @@ class MM_MemoryPoolAddressOrderedList : public MM_MemoryPoolAddressOrderedListBa
 	
 	MM_LargeObjectAllocateStats *_largeObjectCollectorAllocateStats;  /**< Same as _largeObjectAllocateStats except specifically for collector allocates */
 
-	MM_HeapLinkedFreeHeader *_firstUnalignedFreeEntry; /**< it is only for Balanced GC copyforward and non empty survivor region */
-	MM_HeapLinkedFreeHeader *_prevFirstUnalignedFreeEntry;
+	MM_HeapLinkedFreeHeader *_firstCardUnalignedFreeEntry; /**< it is only for Balanced GC copyforward and non empty survivor region */
+	MM_HeapLinkedFreeHeader *_prevCardUnalignedFreeEntry;
+
+	void *_parallelGCAlignmentBase;
+	uintptr_t _parallelGCAlignmentSize;
```

Comment:
add a couple of generic comments like
base address of the region where pool resides